app/Http/Controllers/Api/V1/GoogleClassroomController.php

```diff
@@ -684,4 +684,67 @@ public function publishActivityToGoogleClassroom(Project $project, Playlist $pla
         }
 
     }
+
+
+    /**
+     * To Publish independent activity To Google Classroom under a specific topic
+     *
+     * @urlParam activity required The Id of a independentActivity. Example: 11
+     * @bodyParam access_token string|null The stringified of the GAPI access token JSON object
+     * @bodyParam string course_id (The Google Classroom course id)
```

Comment:
Add "Example: me" for @urlParam, @bodyParam.

Also @param is no available in
https://beyondco.de/docs/laravel-apidoc-generator/getting-started/documenting-your-api
we just use it for our sake with @return